cfme/tests/services/test_catalog_item.py

```diff
@@ -403,7 +404,12 @@ def test_copy_catalog_item():
             2.
             3. Able to copy catalog item
     """
-    pass
+    # copy catalog item
+    new_cat_item = generic_catalog_item.copy()
+    request.addfinalizer(new_cat_item.delete_if_exists)
+
+    # check for catalog item
+    assert new_cat_item.exists
```

Comment:
TODO: I think it would be good to have stronger assertions in this test case, instead of relying so heavily on the `copy` method in the framework.

By this, I mean asserting flash messages (here instead of in the CRUD method), asserting fields in the UI match, and parametrization against types of catalog items.